benchalerts/tests/unit_tests/mocks.py

```diff
@@ -83,6 +84,15 @@ def send(self, *args, **kwargs):
         return MockResponse.from_file(response_path)
 
 
+class MockConbenchClient(ConbenchClient):
+    def __init__(self):
+        super().__init__()
+        self.session.mount("https://", MockAdapter())
```

Comment:
`MockAdapter` appears to have quite some github-flavored implementation. Why is it OK/good to use it in a `MockConbenchClient`?